driver/src/main/java/org/neo4j/driver/internal/async/inbound/InboundMessageDispatcher.java

```diff
@@ -63,6 +66,7 @@ public void queue( ResponseHandler handler )
         else
         {
             handlers.add( handler );
+            updateAutoReadManagingHandlerIfNeeded( handler );
```

Comment:
Hm, I can't really see if there is a problem. However, it is probably incorrect to not clear `autoReadManagingHandler` field because `InboundMessageDispatcher` is pooled together with the channel. It's kind of strange to pool it together with some obsolete auto-read managing handler. Should be fixed in [this commit](https://github.com/neo4j/neo4j-java-driver/pull/528/commits/78cf1e3b5d27ee24862d0ec6811e41d83f3c95dd). Thanks!